manifests/examples/example-app/prometheus-frontend.yaml

```diff
@@ -7,10 +7,9 @@ metadata:
     prometheus: frontend
 spec:
   version: v1.4.1
-  serviceMonitors:
-  - selector:
-      matchLabels:
-        tier: frontend
+  serviceMonitorSelector:
+    matchExpressions:
+    - {key: tier, operator: In, values: [frontend]}
```

Comment:
Any reason for changing to expression matching. In the general case plain label matching seems simpler.